docs/Users_Guide/glossary.rst

```diff
@@ -1822,9 +1822,6 @@ METplus Configuration Glossary
    FCST_MIN_FORECAST
      .. warning:: **DEPRECATED:** Please use :term:`FCST_PCP_COMBINE_MIN_FORECAST`.
 
-   FCST_MAX_FORECAST
```

Comment:
Did you intended to remove this and TIME_SUMMARY_TYPES?  I was just surprised to see them removed since they say they are deprecated already.

Reply:
Yes, these were duplicates that caused warnings that show up sometimes when you build the docs. They don't always show up, so since I saw them this time, I decided to remove them so I didn't forget to later.